lib/kubeclient/aws_eks_credentials.rb

```diff
@@ -20,10 +20,9 @@ def token(credentials, eks_cluster)
         end
         # https://github.com/aws/aws-sdk-ruby/pull/1848
```

Comment:
In this PR the author does end up passing in the region from the sts client dynamically: https://github.com/aws/aws-sdk-ruby/pull/1848/files#diff-f99f43d6d557ede5f15cce411662785f51d791a9c26317db749a4a6bb249e03eR54